Tensile/TensileCreateLibrary.py

```diff
@@ -1099,6 +1100,35 @@ def findLogicFiles(path: Path, logicArchs: Set[str], lazyLoading: bool, experime
 
     return list(str(l) for l in logicFiles)
 
+def createClientConfig(outputPath: Path, masterFile: Path, codeObjectFiles: List[str]) -> None:
+    """Outputs a client config file named best-solution.ini corresponding to a master library 
+       file and code-object parameters created by a TensileCreateLibrary invocation. Also
+       sets best-solution-mode to True.
```

Comment:
I wonder if it would actually be better to refer to this file as a _manifest_ file, instead of a config file. Also, what is the "client" in this context, is it the library implementing Tensile?

Reply:
Config is correct here because it is used to configure the client app which is a stand-in for rocBLAS for testing purposes.